src/aioftp/server.py

```diff
@@ -36,11 +48,8 @@
     "Server",
 )
 
-IS_PY37_PLUS = sys.version_info[:2] >= (3, 7)
-if IS_PY37_PLUS:
-    get_current_task = asyncio.current_task
-else:
-    get_current_task = asyncio.Task.current_task
+
+get_current_task = asyncio.current_task
```

Comment:
Since we have no dispatch, then we do not need this special binding. Just replace `get_current_task` with `asyncio.current_task` everywhere